lib/phases/cold-launch.js

```diff
@@ -166,7 +166,9 @@ ColdLaunch.prototype.testRun = function() {
     .then(function(entry) {
       phase.fullyLoaded = true;
       phase.captureEntryMetadata(entry);
-      phase.device.log.memory(phase.appPid, entry.context);
+      setTimeout(function() {
+        phase.device.log.memory(phase.appPid, entry.context);
+      }, phase.options.memoryDelay);
```

Comment:
Also when waiting for the memoryDelay to expire maybe there should be a debug message? Because right now the last message in the console is "capturing application PID.." and then it stays there for the memoryDelay; looks like it is taking ages to capture the PID. Also if someone introduces a huge delay it might be worthwhile knowing what is up. I wonder if maybe the same thing is true of LAUNCH_DELAY (?)